app/core/utils.coffee

```diff
@@ -698,6 +698,7 @@ getApiClientIdFromEmail = (email) ->
 videoLevels = {
   # gems in the deep
   "54173c90844506ae0195a0b4": {
+    level_name: "Gems in the Deep",
```

Comment:
We should use db level names instead of hard-coding these.  It seems like we'd already have them via course versioning or campaigns collection on this page.